tsconfig.json

```diff
@@ -6,7 +6,7 @@
     "noUnusedParameters": true,
     "strict": true,
     "strictFunctionTypes": false,
-    "module": "commonjs",
+    "module": "esnext",
```

Comment:
This works if we have this, right ?
https://github.com/SonarSource/sonarcloud-core/pull/539/files#diff-d1c6ddbef7eeec60f418b3572311d4daR72

Reply:
Exactly

Reply:
In truth in can work without having this for all browsers that SonarCloud supports, but then we would end up with some duplicate copies of lodash...